### PR TITLE
Static Build and CD for mxnet-cu102/mxnet-cu102mkl

### LINKED ISSUE #17058: CUDA 10.2 Python wheels
## Description
There is currently no cuda 10.2 support for prebuilt Python wheels. (`mxnet-cu102` and `mxnet-cu102mkl`)

It would be greatly appreciated if you could add these versions to your CI.

### PR DESCRIPTION
**Pull request by szha**
## Description ##
add mxnet-cu102/mxnet-cu102mkl, resolves #17058

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [x] add mxnet-cu102 and mxnet-cu102mkl static builds
- [x] add CD job for the new packages.

### CODE REVIEW

**Comment by perdasilva** [comment]
We also should add the ci/docker/Dockerfile.build.ubuntu_gpu_cu102 - for testing and building the dynamic libmxnet ^^

File: ci/docker/Dockerfile.build.ubuntu_gpu_cu102 [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# Dockerfile to run MXNet on Ubuntu 16.04 for GPU
+
+FROM nvidia/cuda:10.1-devel-ubuntu16.04
```
Comment (perdasilva) [inline ci/docker/Dockerfile.build.ubuntu_gpu_cu102]: Need to update the image tag ^^

**Comment by perdasilva** [comment]
I'm waiting for ci-dev to come back up properly so we can run a test =S

**Comment by szha** [comment]
@perdasilva let me know how I can help

**Pull request merged**